packages/front/.env.example

```diff
@@ -3,3 +3,6 @@ VITE_NEAR_NETWORK=
 VITE_NEAR_NODE_URL=
 VITE_API_GRAPHQL_URL=
 VITE_GTM_ID=
+VITE_RELAYER_URL=
+VITE_RELAYER_ACCOUNT=
```

Comment:
VITE_RELAYER_ACCOUNT and VITE_RELAYER_FEE are not used since this data can be pulled from relayer - remove from .env.example